third_party/xla_client/topology.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
```

Comment:
I don't think we should automatically decide to move topology.proto to torch_ala. Is this for verification purposes?

Reply:
copied this line to bypass this problem, and this line is used to define the syntax is `proto3`, because format requirement;

not for verification purpose;